docs/scaffold-a-new-project/README.md

```diff
@@ -0,0 +1,72 @@
+# Scaffold a New Project
+
+Neutrino provides a scaffolding tool to help you quickly jump-start a project. `create-neutrino-project` uses
+presets behind the scene to build projects. If you are not familiarized with them, take a moment to
+explore [what presets are](https://neutrino.js.org/presets/).
+
+## Getting Started
+Run the following command to start the interactive session. Substitute `<directory-name>` with the directory name you wish to create for this project.
+
+### Yarn
+
+```
+❯ yarn create neutrino-project <directory-name>
+```
+
+_Note: The `create` command is a shorthand that helps you do two things at once. See the [Yarn create docs](https://yarnpkg.com/lang/en/docs/cli/create/) for more details._
+
+### npm
+
+```
+❯ npm install -g create-neutrino-project
+❯ create-neutrino-project <directory-name>
+```
+
+## Available Projects
+`create-neutrino-project` presently offers a scaffolding project to build an application, a library,
+or components. Depending on the project type, the CLI helper may offer different flavors
+of that project to scaffold. Each project type harnesses the power of a preset to configure itself.
+
+| Project     | Project Type     | Preset                                                                                                             |
+|-------------|------------------|--------------------------------------------------------------------------------------------------------------------|
+| Application | React            | [`neutrino-preset-react`](https://neutrino.js.org/presets/neutrino-preset-react/)                                  |
+| Application | Web              | [`neutrino-preset-web`](https://neutrino.js.org/presets/neutrino-preset-web/)                                      |
+| Application | Node.js          | [`neutrino-preset-node`](https://neutrino.js.org/presets/neutrino-preset-node/)                                    |
+| Library     | Web              | [`neutrino-preset-taskcluster-web-library`](https://www.npmjs.com/package/neutrino-preset-taskcluster-web-library) |
+| Components  | React Components | [`neutrino-preset-react-components`](https://www.npmjs.com/package/neutrino-preset-react-components)               |
+
+## Test Runners
+Testing is necessary because we all make mistakes. `create-neutrino-project` offers the following test runners:
```

Comment:
Even though we all make mistakes, we don't want to use that as the motivator to use testing. Leave the decision up to the user whether they want to use it, and avoid connotations that may shame them into using a test runner. 😆 

"If you wish to use a test runner, Neutrino will offer to set one up for you during the scaffolding phase."